rclcpp/CMakeLists.txt

```diff
@@ -17,7 +18,9 @@ else()
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra -Wpedantic")
 endif()
 
-include_directories(include)
+include_directories(
+  include
+  ${builtin_interfaces_INCLUDE_DIRS})
```

Comment:
> you mean on the packages which use this header file?

No I mean on the targets which use the header file.

> As this is a header-only implementation, there is no target inside rclcpp which uses this header file.

If no target needs these header files then it doesn't need to be in the `include_directories()`. I would have thought a test used this header file in a compilation unit in this package.